streams/examples/src/main/java/org/apache/kafka/streams/examples/wordcount/WordCountDemo.java

```diff
@@ -82,6 +81,23 @@ public static void main(String[] args) throws Exception {
         // need to override value serde to Long type
         counts.to(Serdes.String(), Serdes.Long(), "streams-wordcount-output");
 
+        GlobalKTable<String, Long> gTable = builder.globalTable("topicN", "storeN");
+
+        source.join(gTable, new KeyValueMapper<String, String, String>() {
+            @Override
+            public String apply(String key, String value) {
+                return key;
+            }
+        }, new ValueJoiner<String, Long, String>() {
+            @Override
+            public String apply(final String value1, final Long value2) {
+                return value1 + value2;
+            }
+        });
+
+        System.out.println(builder.build().describe());
```

Comment:
Should we to a `final Topology topology = builder.build()` and use `topology` here and in `KafkaStreams` constructor?